src/pkg/fault/item.go

```diff
@@ -15,6 +15,7 @@ type itemType string
 
 const (
 	FileType          itemType = "file"
+	AttachmentType    itemType = "attachment"
```

Comment:
I'm not entirely confident about the choice of "attachment" as a type here, since it's specific to certain types of data and not others.  That sets us up to have N different types, instead of one type with N different identifiers.  For example, if this were, I dunno, an `ItemPropertyType`, with a category of `Attachment`, then we only need one interface for all ItemProperty skips, all of which can dynamically declare different categories.  Easier to extend and maintain that way.  (similary, we should probably change "fileType" to "itemType", one category of which can be a File.)

Not entirely sure what to call it.  Maybe "attachment" is sufficient for the short term?  Don't want to block on this, since we should be able to rename in a fast follow without much fuss.  But do help me think on the best name, please.